_episodes/08-putting-it-all-together.md

```diff
@@ -146,25 +146,26 @@ styles and the source codes that create them.
 
 ### `plt` pyplot versus object-based matplotlib
 
-Matplotlib integrates nicely with the Numpy package and can use Numpy arrays
+Matplotlib integrates nicely with the NumPy package and can use NumPy arrays
 as input of the available plot functions. Consider the following example data,
-created with Numpy:
+created with NumPy by drawing 1000 samples from a normal distribution with mean 0 and
+standard deviation 0.1:
 
 ~~~
 import numpy
-x = numpy.linspace(0, 5, 10)
-y = x ** 2
+s = numpy.random.normal(0, 0.1, 1000)
+
 ~~~
 {: .language-python}
 
-To make a scatter plot of `x` and `y`, we can use the `plot` command directly:
+To make a histogram plot of our draws from the normal distribution, we can use the `hist` function directly:
```

Comment:
`To make a histogram plot` is fine ... but `To plot a histogram` seems simpler, no? Feel free to ignore this suggestion...

Do you think we should emphasize once again that `plt` is `matplotlib.pyplot`?

> we can use the `hist` function provided by `matplotlib.pyplot` directly:

Reply:
without that clarification, I'm not sure we need `directly` at the end.

Reply:
I like it because it calls attention to the fact that it isn't `kind=hist`, or similar. But I also don't feel _that_ strongly.